tests/scripts/run-throughput-perf.py

```diff
@@ -290,7 +329,7 @@ def main(args):
                 shutil.copy(csv_file_name, clr_root)
 
                 # For each benchmark, call measurement.py
-                measurement_args = ["py",
+                measurement_args = [python_exe,
```

Comment:
This is something I guess I did not notice last time, but why do we check to see if benchview_path is None before making the call to upload, but not before using measurement.py?  Seems we should have the same check here as well.